### PR TITLE
[7.0] Add module 'partner_address_street_3'

### PR DESCRIPTION
**Pull request by gurneyalex**
backport the addon from 8.0

rebased #60

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Two PEP8 issues:

```
./partner_address_street3/model/partner.py:42:5: E303 too many blank lines (2)
./partner_address_street3/model/partner.py:60:1: W391 blank line at end of file
```

A README.rst file is missing.

**Comment by gurneyalex** [comment]
failure on runbot is caused by unrelated warnings.

File: partner_address_street3/README.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Firstname Lastname <email.address@example.org>
```
Comment (antespi) [inline partner_address_street3/README.rst]: Who are the contributors?

File: partner_address_street3/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Street3 in addresses',
+    'version': '0.1',
```
Comment (antespi) [inline partner_address_street3/__openerp__.py]: 7.0.1.0.0

**Comment by antespi** [comment]
:+1: After minor remarks fixed

File: partner_address_street3/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    'author': 'Camptocamp',
```
Comment (RoelAdriaans) [inline partner_address_street3/__openerp__.py]: Add OCA as author so it shows up in the odoo app store

File: partner_address_street3/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Street3 in addresses',
+    'version': '7.0.0.1.0',
```
Comment (pedrobaeza) [inline partner_address_street3/__openerp__.py:23]: Shouldn't this be 7.0.1.0.0?

Comment (gurneyalex) [inline partner_address_street3/__openerp__.py:23]: I'm using the same number as the 8.0 version from which this was ported.

Comment (pedrobaeza) [inline partner_address_street3/__openerp__.py:23]: Well, OK, usually the first version arriving to OCA has the 1.0.0 version milestone, but not a blocking issue.

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**